src/amo/components/Addon/index.js

```diff
@@ -177,15 +184,29 @@ export class AddonBase extends React.Component {
       isPreviewingTheme,
       installStatus,
     } = this.props;
-    const previewURL = addon ? addon.previewURL : null;
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
-    const iconUrl = getAddonIconUrl(addon);
```

Comment:
since it's only needed by the code below...

it also came up in a preview [PR review](https://github.com/mozilla/addons-frontend/pull/4914#discussion_r187984430)